src/components/pathway-progress/PathwayNode.jsx

```diff
@@ -6,6 +6,7 @@ import capitalize from 'lodash.capitalize';
 
 import { CONTENT_TYPES, IN_PROGRESS } from './constants';
 import { shortenString } from '../course/data/utils';
+import { linkToCourse } from '../skills-quiz/data/utils';
```

Comment:
We may want to consider abstracting the `linkToCourse` function outside of the `skills-quiz` module if we intend it to be a generic utility function that can be used across the app and outside of the skills quiz feature.

Reply:
Good call-out, moved to src/components/course/data/utils.jsx